### PR TITLE
fix: remove unneccessary dependencies

### PR DESCRIPTION
**Pull request by mpetrunic**
Can we remove private key signing altogether or move it to different package?

## Related Issue Or Context

Closes: #<issue>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Comment by MakMuftic** [comment]
Remove it altogether, as fee oracle is the one signing this information - never SDK. I guess this flow is here from the time that our local setup with fee oracle was still being worked on - @wainola @enemycnt please confirm this.

File: packages/sdk/package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
 	"author": "Sygmaprotocol Product Team",
 	"license": "LGPL-3.0-or-later",
 	"devDependencies": {
+		"@chainsafe/chainbridge-contracts": "2.1.4",
```
Comment (enemycnt) [inline packages/sdk/package.json:34]: We are using `@buildwithsygma/sygma-contracts` package here. Do you have specific reason to add `@chainsafe/chainbridge-contracts`?

**Review by enemycnt — CHANGES_REQUESTED** [review]

File: packages/sdk/package.json (hunk 1 above)
Comment (mpetrunic) [reply packages/sdk/package.json:34]: https://github.com/sygmaprotocol/sygma-sdk/blob/main/packages/sdk/integration/test/utils.test.ts#L3

Comment (mpetrunic) [reply packages/sdk/package.json:34]: Note that this is dev dependency, I haven't remove prod dep^^

**Review by enemycnt — APPROVED** [review]

**Review by enemycnt — APPROVED** [review]

**Review by MakMuftic — APPROVED** [review]

**Pull request merged**